__tests__/serializers/slateToHtml/index.spec.ts

```diff
@@ -122,3 +122,111 @@ describe('empty content', () => {
     })
   })
 })
+
+
+describe('line breaks', () => {
+  it('does nothing with a line break if convertLineBreakToBr is false', () => {
+    const slate: any[] = [
+      {
+        children: [
+          {
+            text: 'Paragraph with \n line',
+          },
+          {
+            text: '\n',
+          },
+          {
+            text: '\n',
+          },
+          {
+            text: 'breaks.',
+          },
+        ],
+        type: 'p',
+      },
+    ]
+    const html = '<p>Paragraph with \n line\n\nbreaks.</p>'
+    expect(slateToHtml(slate)).toEqual(html)
+  })
+
+  it('adds br tag between text elements', () => {
+    const slate: any[] = [
+      {
+        children: [
+          {
+            text: 'Paragraph 1',
+          },
+        ],
+      },
+      {
+        children: [
+          {
+            text: 'Paragraph 2',
+          },
+        ],
+      },
+    ]
+    const html = 'Paragraph 1<br>Paragraph 2'
+    expect(slateToHtml(slate, { ...slateToDomConfig, convertLineBreakToBr: true })).toEqual(html)
+  })
```

Comment:
I've given this some thought - seems like the most logical way to deal with this scenario. Thanks for adding it! I can see it involves some additional logic in the serializer.